aten/src/ATen/native/cuda/TensorTransformations.cu

```diff
@@ -41,10 +41,17 @@ kernel_pointwise_flip_apply2(const cuda::detail::TensorInfo<scalar_t, IndexType>
 }
 
 template <typename scalar_t>
-__global__
-void flip_cuda_kernel(scalar_t* in_tensor, scalar_t* out_tensor, int64_t N, int64_t* flip_dims, int64_t flip_dims_size,
-                      int64_t* strides, int64_t* strides_contiguous, int64_t* shape, int64_t total_dims) {
-
+C10_LAUNCH_BOUNDS_1(AT_APPLY_THREADS_PER_BLOCK)
```

Comment:
why `AT_APPLY_THREADS_PER_BLOCK` instead of `cuda::getApplyBlockSize()`?

Reply:
So so this is a local AT_APPLY_THREADS_PER_BLOCK, likely because the original is in the anonymous namespace. Originally, I tried to leave as much alone as I could, but now that you mention it, I've replaced the use of AT_APPLY...

Reply:
(Generally, it's a bit hard to decide the scope, one could probably make it into a largeish cleanup project, but there are 60 other files with kernels without launch bounds, so I'd look at those first...)